modules/nw-metallb-configure-bgp-advertisement.adoc

```diff
@@ -0,0 +1,164 @@
+:_content-type: PROCEDURE
+[id="nw-metallb-configure-BGP-advertisement_{context}"]
+= Configuring MetalLB with a BGP advertisement
+
+Configure MetalLB as follows so that the `IPAddressPool` is advertised with the BGP protocol.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+== Example: Advertise a basic address pool configuration with BGP
+
+In the following example, the peer BGP routers receive one `203.0.113.200/32` route and one `fc00:f853:ccd:e799::1/128` route for each load-balancer IP address that MetalLB assigns to a service.
+Because the `localPref` and `communities` fields are not specified, the routes are advertised with `localPref` set to zero and no BGP communities.
+
+.Procedure
+
+. Create an IP address pool.
+
+.. Create a file, such as `ipaddresspool.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  namespace: metallb-system
+  name: doc-example-bgp-basic
+spec:
+  addresses:
+    - 203.0.113.200/30
+    - fc00:f853:ccd:e799::/124
+----
+
+.. Apply the configuration for the IP address pool:
++
+[source,terminal]
+----
+$ oc apply -f ipaddresspool.yaml
+----
+
+. Create a BGP advertisement.
+
+.. Create a file, such as `bgpadvertisement.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: BGPAdvertisement
+metadata:
+  name: bgpadvertisement-basic
+  namespace: metallb-system
+spec:
+  ipAddressPools:
+  - doc-example-bgp-basic
+----
+
+.. Apply the configuration:
++
+[source,terminal]
+----
+$ oc apply -f bgpadvertisement.yaml
+----
+
+== Example: Advertise an advanced address pool configuration with BGP
```

Comment:
missing anchor ID per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings